CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 
 ### Fixed
 
+- Fixed the new view changes affecting some weapons by moving it to the GM:CalcView hook and adding a new break condition (by @TimGoll)
```

Comment:
```suggestion
- Fixed the new view changes preventing weapons from modifying the playerview (by @TimGoll)
- Fixed the new view changes affecting non-player entities (by @TimGoll)
```

is `playerview` an accurate description?
My main point is that these are 2 related but different fixes
Feel free to change my wording

Reply:
yeah, that's fine!